src/build/index.ts

```diff
@@ -0,0 +1,60 @@
+import path from "node:path";
+import process from "node:process";
+import fs from "node:fs/promises";
+import { type RollupOptions, rollup } from "rollup";
+
+import { JspmError, exists } from "../utils";
+import type { Flags } from "../types";
+import { RollupImportmapPlugin } from "./rollup-importmap-plugin";
+
+export default async function build(flags: Flags) {
+  if (!flags.entry && !flags.buildConfig) {
+    throw new JspmError(`Please provide entry for the build`);
+  }
+
+  let buildConfig: RollupOptions;
+  let outputOptions: RollupOptions["output"];
+
+  if (flags.entry) {
```

Comment:
What happens if entry is not provided? Surely we always need one?

Reply:
We don't need the `--entry` all the time. As they can export `input` from the `config` file. So, when build config is passed. Rollup takes the entry from it.

Reply:
Thanks for clarifying. I would still prefer to make these optional arguments over a flag.